taxi/tests/test_views.py

```diff
@@ -0,0 +1,144 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+
+from taxi.models import Manufacturer, Driver, Car
+from taxi.forms import DriverSearchForm, ManufacturerSearchForm, CarSearchForm
+
+MANUFACTURER_URL = reverse("taxi:manufacturer-list")
+DRIVER_URL = reverse("taxi:driver-list")
+CAR_URL = reverse("taxi:car-list")
+
+
+class PublicTest(TestCase):
+    """Test the publicly available pages"""
+    def test_login_required_for_manufacturer_list(self) -> None:
+        """Test the publicly available manufacturer pages"""
+        res = self.client.get(MANUFACTURER_URL)
+
+        self.assertNotEqual(res.status_code, 200)
+
+    def test_login_required_for_driver_list(self) -> None:
+        """Test the publicly available driver pages"""
+        res = self.client.get(DRIVER_URL)
+
+        self.assertNotEqual(res.status_code, 200)
+
+    def test_login_required_for_car_list(self) -> None:
+        """Test the publicly available car pages"""
+        res = self.client.get(CAR_URL)
+
+        self.assertNotEqual(res.status_code, 200)

```

Comment:
check if status code is equal to expected instead. You can also check if it redirects to expected page